### PR TITLE
racket: update to 6.4

### PR DESCRIPTION
**Pull request by Silverbeard00**

### CODE REVIEW

File: srcpkgs/racket/template [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-revision=3
+version=6.4
+revision=1
 build_style=gnu-configure
 build_wrksrc=src
 hostmakedepends="gsfonts"
-makedepends="sqlite-devel gtk+-devel"
-depends="gtk+"
+makedepends="sqlite-devel gtk+3-devel"
+depends="gtk+3"
```
Comment (xtraeme) [inline srcpkgs/racket/template]: This depends line is unnecessary, remove it.

Comment (Silverbeard00) [inline srcpkgs/racket/template]: Before I submit another incorrect template are you refering to "makedepends", "depends" or do you mean remove both?

Comment (Silverbeard00) [inline srcpkgs/racket/template]: Failed to build with both removed so I'm assuming you mean the _gtk+3_ dependancy

I'll remove that and try again

**Comment by Silverbeard00** [comment]
I can't help but feel I screwed something up.  It's saying the build system is pending for the old template and the new one with the changes xtraeme wanted.

I'll be reading up on how to use git properly so it doesn't happen again.

**Comment by Silverbeard00** [comment]
Builds as 64bit using _xbps-src_ on my computer.  But in the travis-ci it fails to build the x86_64 but successfully completes the 32bit version.

I'm not sure if this is an issue with the build system.  Looking into it tonight.

**Comment by Duncaen** [comment]
You have to remove the old commits and force push to your branch.
travis is building your old commits first.

**Comment by Silverbeard00** [comment]
Alright, well technically it's working on ARM now.  There's a racket error on the ARMv7 version but it still creates the package.  What's weird is the same error pops up on ARMv6 but that one passes the travis-ci check somehow.

**Comment by Silverbeard00** [comment]
```
armv7l-linux-gnueabihf-objdump: /destdir/armv7l-linux-gnueabihf/racket-6.4/usr/bin/mred: Bad value

armv7l-linux-gnueabihf-objdump: /destdir/armv7l-linux-gnueabihf/racket-6.4/usr/bin/mzscheme: Bad value
```

Here's the error it's giving.  It still compiles the package for ARM v6 & v7 (only errored on the v7 for some reason).  I'm posting this incase someone knows what it is.  

Compiling fine on x86 and x86_64 so it must be an ARM specific issue.  

```

=> Creating racket-6.4_1.armv7l.xbps for repository /host/binpkgs ...

=> racket-6.4_1: running post-pkg hook: 00-register-pkg ...

=> Registering racket-6.4_1.armv7l.xbps into /host/binpkgs ...

index: added `racket-6.4_1' (armv7l).

index: 1 packages registered.
```

**Comment by Duncaen** [comment]
Please fix the commit history first.

```
=> racket-6.4_1: running do_configure ...
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking target system type... x86_64-unknown-linux-gnu
=== Documentation build disabled
checking for x86_64-unknown-linux-gnu-gcc... cc
```
The target system and the compiler `cc` are definitively wrong.

**Comment by Silverbeard00** [comment]
Wow I totally missed that.  I'll look into that then and try to get the commit history fixed.  Thanks.

**Comment by thypon** [comment]
Updated by myself. I added also a working update script.

**Pull request closed without merging**